packages/test-data/granules/disco_granules_duplicate_fake_granules/granule-4.nc

```diff
@@ -0,0 +1 @@
+This is granule-1.nc
```

Comment:
I just realized when I moved these wholesale, their contents are .... misleading?   I'm not sure I care about regenning .md5s, but it feels untidy.

Reply:
Sorted. 044a011f3